packages/types/src/http/httpHandlerInitialization.ts

```diff
@@ -90,4 +90,9 @@ export interface FetchHttpHandlerOptions {
    * these limitations before enabling keepalive.
    */
   keepAlive?: boolean;
+
+    /** 
+     * A string indicating whether credentials will be sent with the request always, never, or only when sent to a same-origin URL. Sets request's credentials.
+     */
+  credentials?: RequestCredentials;
```

Comment:
```suggestion
  credentials?: "include" | "omit" | "same-origin"
```

to remove dependency on lib.dom types.